assets/css/style.css

```diff
@@ -103,6 +103,144 @@ html[dir='rtl'] .clearButton {
     padding-bottom: 10px;
 }
 
+.posRelative {
+    position: relative;
+}
+
+#apertiumLogo {
+    margin-bottom: -.25em;
+}
+
+#localeSelectDiv {
+    width: 35%;
+}
+
+#localeGlobeIcon {
+    padding: 5px 5px 0 0;
+}
+
+#srcLangSelect,
+#dstLangSelect {
+    display: inline-block !important; /* sass-lint:disable-line no-important */
+}
+
+#cancelTranslateWebpage {
+    display: none;
+}
+
+#translateDoc {
```

Comment:
This rule along with all the file stuff should probably be in the translation section (there's another issue to split all this CSS into separate files, I believe)